main/EDTypesMod.F90

```diff
@@ -14,27 +14,35 @@ module EDTypesMod
   use FatesConstantsMod,     only : n_anthro_disturbance_categories
 
   implicit none
+  private               ! By default everything is private
   save
 
-  integer, parameter :: maxPatchesPerSite  = 14   ! maximum number of patches to live on a site
-  integer, parameter :: maxPatchesPerSite_by_disttype(n_anthro_disturbance_categories)  = &
+  ! Make public necessary subroutines and functions
+  public :: val_check_ed_vars
```

Comment:
In the discussion at the top of the PR, you mentioned that you were moving procedure specifications to the end after the module variables were defined.  I have no strong feelings about order, but wanted to note it here.

Reply:
Good catch.  I'll fix that.